src/preflightCheck.js

```diff
@@ -33,19 +33,19 @@ export default function createPreflightCheck () {
 				check = transform( inputCode, options ).code;
 			}
 
-			if ( ~check.indexOf( 'import _inherits' ) ) helpers = RUNTIME;
-			else if ( ~check.indexOf( 'function _inherits' ) ) helpers = INLINE;
-			else if ( ~check.indexOf( 'babelHelpers' ) ) helpers = EXTERNAL;
-			else {
-				ctx.error( 'An unexpected situation arose. Please raise an issue at https://github.com/rollup/rollup-plugin-babel/issues. Thanks!' );
-			}
-
 			if (
 				!~check.indexOf( 'export default' ) &&
 				!~check.indexOf( 'export default Foo' ) &&
 				!~check.indexOf( 'export { Foo as default }' )
 			) {
-				ctx.error( 'It looks like your Babel configuration specifies a module transformer. Please disable it. See https://github.com/rollup/rollup-plugin-babel#configuring-babel for more information' );
+				ctx.error( 'Rollup requires that your Babel configuration keeps ES6 module syntax intact. Unfortunately it looks like your configuration specifies a module transformer to replace ES6 modules with CommonJS. To continue you have to disable it. Most commonly it\'s a CommonJS transform added by @babel/preset-env - in such case you should disable it by adding `modules: false` option to that preset (described in more detail here - https://github.com/rollup/rollup-plugin-babel#modules ).' );
```

Comment:
> it looks like your configuration specifies a module transformer to replace ES6 modules with CommonJS

Technically it doesn't have to be CommonJS, it might be some other module format - UMD, AMD or System. In this particular sentence I would describe it as "with other module format"

I would also insert a new empty line before "Most commonly..." for readability.

Also because it's such a long message I would prefer hoisting it to the top and using a variable here - smth like `const TRANSFORMED_MODULES_ERROR = "..."`

Reply:
You are right! I rework that part.